benchmark/benchmark17_test.go

```diff
@@ -0,0 +1,57 @@
+// +build go1.7
+
+package benchmark
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/benchmark/stats"
+)
+
+func BenchClient(b *testing.B, maxConcurrentCalls, reqSize, respSize int, runMode, enableTrace string) {
+	if strings.EqualFold(enableTrace, "Tracing") {
+		grpc.EnableTracing = true
+	} else {
+		grpc.EnableTracing = false
+	}
+	if strings.EqualFold(runMode, "Unary") {
+		runUnary(b, maxConcurrentCalls, reqSize, respSize)
+	} else {
+		runStream(b, maxConcurrentCalls, reqSize, respSize)
+	}
+}
+
+func BenchmarkClient(b *testing.B) {
+	modeTrace := []struct {
+		runMode     string
```

Comment:
Please use consts instead of string literals:

```go
const (
  runModeUnary = "Unary"
  runModeStreaming = "Stream"
)

switch runMode {
  case runModeUnary:
    ...
}
```